Tribler/Core/Modules/restapi/trustview_endpoint.py

```diff
@@ -18,6 +18,14 @@
 MAX_PEERS = 500
 MAX_TRANSACTIONS = 2500
 
+networkx_version = float('.'.join(nx.__version__.split('.')[:2]))
```

Comment:
Why not use `LooseVersion`?

Reply:
There is no need to check for patch version in networkx so float comparison was sufficient. For consistency, you're right, it is a good idea to use LooseVersion. I have updated the PR to use LooseVersion.